src/api-specification/wallet-controller/retrieving-token-balance.md

```diff
@@ -0,0 +1,69 @@
+# Retrieving token balance
+
+> Retrieves the token balance on the [authorized FIB user's](../auth-controller/authorizing-a-user-in-the-system.md) account at the time of the request.
+
+**Protocol**: `HTTP`
+
+**Encoding**: `JSON`
+
+**Endpoint**: `/data/api/v1/wallet-management/balance`
+
+**Method**: `GET`
+
+**Header**: `Authorization` — represented by a `string` that is the `accessToken` value that was returned either at the authorization, or the previous time the token [has been refreshed](refreshing-authentication-tokens.md).
+
+#### Input/request structure
+
+A `GET` request to the endpoint with the `Authorization` header specified.
+
+#### Output/response structure
+
+::: details Show
+
+::: code-group Data structure
+
+```json5 [Structure]
+{
+  "status": {
+    "code": integer($int32),
+    "name": "string",
+    "message": "string"
+  },
+  "data": {
+    "tokenId": {
+      "definitionId": "assetName#assetDomain",
+      "accountId": "accountName@accountDomain"
+    },
+    "balance": integer($int64)
+  }
+}
+```
+
+```json5 [Example]
+{
+  "status": {
+    "code": 200,
+    "name": "OK",
+    "message": "Token balance has been retrieved successfully"
+  },
+  "data": {
+    "tokenId": {
+      "definitionId": "credit#admin",
+      "accountId": "admin@admin"
+    },
+    "balance": 500
+  }
+}
+```
+
+:::
+
+:::
+
+### Responses
+
+| Response Code | Description                                    |
+| :-----------: | ---------------------------------------------- |
+| `200`         | Token balance has been retrieved successfully. |
+| `401`         | ???                                            |
```

Comment:
403 may return as well - Forbidden (in case a non-admin peer attempts to request the credit balance of another peer)